ClientApp/src/components/UserProfile.js

```diff
@@ -6,22 +6,6 @@ import { changeCurUserInfo } from '../scripts/requests.js';
 
 import './UserProfile.css';
 
-//function getProfile(userId) {
```

Comment:
Personally I would have kept this ....

Reply:
I'm sorry but Granny Bample has asked that her personal information be removed from all Artefactor services and databases. Something about our "total incompetence" and "blatant disregard for people who like family trees".